openstack/cronus/templates/rhea/_config.yaml.tpl

```diff
@@ -19,6 +19,23 @@ rhea:
     endpointType: {{ .Values.config.endpointType }}
 {{- end }}
   auth:
+    keystone:
+{{- if .Values.config.keystone }}
+{{- range $key, $value := .Values.config.keystone }}
+    {{- if $value }}
+      {{ $key }}: {{ $value }}
+    {{- end }}
+{{- end }}
+{{- if .Values.global.cronus_service_password }}
+      password: {{ .Values.global.cronus_service_password }}
+{{- end }}
+{{ else }}
+      authUrl: {{ .Values.config.authUrl }}
```

Comment:
Unfortunately the authUrl is not currently in the globals. I reached out asking, and it was mentioned that it would be added. Maybe I can make a PR this week

Reply:
Thanks, Corey!